### ISSUE TITLE
Accessibility project

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/product-backlog#1661]
To see all of the accessibility issues tracked by Product see: https://github.com/orgs/hypothesis/projects/144/views/9

These are Issues related to our accessibility evaluation by OCAD (leading to our VPAT):
- https://github.com/hypothesis/client/issues/6185
- https://github.com/hypothesis/client/issues/6306
- https://github.com/hypothesis/client/issues/6307
- https://github.com/hypothesis/product-backlog/issues/1660
- https://github.com/hypothesis/product-backlog/issues/1659

In addition, we received feedback from a book publisher on accessibility in our tool ([Slack thread](https://hypothes-is.slack.com/archives/C8TPC8XMK/p1736535764817829)). We should think of these as nice to haves, not required.

- https://github.com/hypothesis/product-backlog/issues/1633
- https://github.com/hypothesis/product-backlog/issues/1634
- https://github.com/hypothesis/product-backlog/issues/1635
- https://github.com/hypothesis/product-backlog/issues/1636
- https://github.com/hypothesis/product-backlog/issues/1637
- https://github.com/hypothesis/product-backlog/issues/1638
- https://github.com/hypothesis/product-backlog/issues/1639
